core/io/resource_format_binary.cpp

```diff
@@ -1399,7 +1399,7 @@ Error ResourceFormatLoaderBinary::rename_dependencies(const String &p_path, cons
 		}
 
 		if (p_map.has(path)) {
-			String np = p_map[path];
+			const String &np = p_map[path];
```

Comment:
I'd also replace any cases where the indirection is used exactly once, and where the type conversion is clear

This applies when the target type (or function argument) is a fixed type, or when the source (like if it is a `HashMap`) is clear (and the types match)

I.e. when the conversion isn't a cast